vllm/scripts.py

```diff
@@ -24,6 +24,11 @@ def signal_handler(sig, frame):
 
 
 def serve(args: argparse.Namespace) -> None:
+    if args.model is not None:
```

Comment:
Oh actually this might always default to "facebook/opt-125m"

Reply:
Fixed